simulation/tests/test_collectors.py

```diff
@@ -4,9 +4,20 @@
 from simulation.model import CnzAgentBasedModel
 
 
+def model_factory(**model_attributes):
```

Comment:
I like the idea of a model factory, and agree we should use it more generally across the tests.